partners_api/partners_api_tests/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+project(partners_api_tests)
+
+add_definitions(-DOMIM_UNIT_TEST_WITH_QT_EVENT_LOOP)
+
+set(
+  SRC
+  booking_tests.cpp
+  uber_tests.cpp
+)
+
+omim_add_test(${PROJECT_NAME} ${SRC})
+
+omim_link_libraries(
+  ${PROJECT_NAME}
+  partners_api
+  platform
+  coding
+  base
+  tomcrypt
+  jansson
+  stats_client
+  ${LIBZ}
+  ${Qt5Core_LIBRARIES}
+)
+
+if (PLATFORM_MAC)
+  omim_link_libraries(
+    ${PROJECT_NAME}
+    ${Qt5Widgets_LIBRARIES}
```

Comment:
As far as I know @syershov insists that we need some kind of macro (or function) omim_link_with_platform(), what will include all needed frameworks and Qt5 parts.